src/test/librbd/fsx.cc

```diff
@@ -2028,7 +2022,7 @@ main(int argc, char **argv)
 			break;
 		case 'l':
 			maxfilelen = getnum(optarg, &endp);
-			if (maxfilelen <= 0)
+			if (maxfilelen == 0)
```

Comment:
getnum() is capable of returning <0, as it calls strtol. Why are you changing this comparison?

Also, we copied this from...somewhere...and it might be polite to send any fixes to whatever groups are doing real maintenance of fsx at this point. :)